src/ocnicepost.fd/masking_mod.F90

```diff
@@ -27,7 +27,7 @@ subroutine remap_masks(vfill)
     ! local variables
     integer            :: rc, ncid, varid, n
     character(len=240) :: wgtsfile
-    real               :: minlat = -79.75
+    real               :: minlat = -78.00
```

Comment:
@AminIlia-NOAA Is this minimum valid for all OCN resolutions?

Reply:
Thank you for our comment @DeniseWorthen . It is very good point. 
I think it is the best to move it to -78.63 so we don't loose any computational points in 1/4 deg. We may still have spurious results in 5 deg resolution, however, I believe 5deg resolution has no application in today's world.

I committed a change for minlat=-78.63.

Reply:
@DeniseWorthen Thank you for your comment. 

Yes, this should resolve it. However as I mention the problem could be from another thing. We will loose some computational nodes for 1/4 deg however those are outside ocean edge. Please let me know your opinion.